src/three-components/Renderer.js

```diff
@@ -45,9 +45,10 @@ export default class Renderer extends EventDispatcher {
     const webGlOptions = {antialias: true};
 
     // Only enable certain options when Web XR capabilities are detected:
-    if (IS_AR_CANDIDATE) {
+    // TODO: hook to transparent attribute
+    // if (IS_AR_CANDIDATE) {
```

Comment:
Was this removed intentionally, or just a vestigial comment?

Reply:
I was reusing the {alpha: true, preserveDrawingBuffer: true} in the next line there... This might make it more clear: f4f779c09f0b12d5bffaafe2ca283a6edbeb0e1e